### ISSUE TITLE
Finish gateway configuration of control panel.

### ISSUE THREAD

**Issue by virtadpt** [Byzantium/Byzantium#8]
Could somebody pick up development of the gateway configuration sub-app (gateways.py)?  I just realized that I might know more about the manual installation process at this point, and should spend time documenting that so that the .iso build process can be finished.  I've based the code off of networkconfiguration.py and I'm re-using some of the HTML templates as the user interface.

I have design notes that I can add to this ticket for whomever wants to pick it up.

Please?  Anybody?

**Comment by virtadpt** [comment]
Gateway mode was tested at the USA Science and Engineering Festival this weekend because the convention center charges usurious rates for wi-fi.  I uncovered a few things in the process:

Tethering to cellular phones is almost seamless.  We plugged ag4ve's phone into a USB jack, it was detected automatically and the network interface usb0 was created.  dhcpcd was run on usb0 and it was given network configuration information per usual.  Connectivity was tested and it worked.

I'm going to open some sub-tickets in the bug tracker and reference them off of this meta-ticket.
